### PR TITLE
dev-db/mongodb: bump to 4.1.6 / app-admin/mongo-tools: bump to 4.1.6

### PR DESCRIPTION
**Pull request by lKk2**
Signed-off-by: Vitor Ferrari <vitor@sulvo.com>
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

**Comment by ultrabug** [comment]
thanks @lKk2 !

ping @hydrapolic for review plz ?

**Comment by hydrapolic** [comment]
Sure, i'll check, but away from computer until friday.

File: dev-db/mongodb/mongodb-4.1.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+	"${FILESDIR}/${PN}-3.4.7-no-boost-check.patch"
+	"${FILESDIR}/${PN}-3.6.1-fix-scons.patch"
+	"${FILESDIR}/${PN}-4.0.0-no-compass.patch"
+)
+
+S=${WORKDIR}/${MY_P}
+
+pkg_pretend() {
+	if [[ -n ${REPLACING_VERSIONS} ]]; then
+		if ver_test "$REPLACING_VERSIONS" -lt 3.6; then
```
Comment (hydrapolic) [inline dev-db/mongodb/mongodb-4.1.6.ebuild]: Please read the release notes at https://docs.mongodb.com/master/release-notes/4.2/ and modify this section.

**Comment by hydrapolic** [comment]
mongo-tools fails to build:
```
>>> Emerging (1 of 2) app-admin/mongo-tools-4.1.6::local
 * Fetching files in the background.
 * To view fetch progress, run in another terminal:
 * tail -f /var/log/emerge-fetch.log
 * mongo-tools-4.1.6.tar.gz BLAKE2B SHA512 size ;-) ...                                                                                                                                                                [ ok ]
>>> Unpacking source...
>>> Unpacking mongo-tools-4.1.6.tar.gz to /var/tmp/portage/app-admin/mongo-tools-4.1.6/work
>>> Source unpacked in /var/tmp/portage/app-admin/mongo-tools-4.1.6/work
>>> Preparing source in /var/tmp/portage/app-admin/mongo-tools-4.1.6/work/mongo-tools-r4.1.6 ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/app-admin/mongo-tools-4.1.6/work/mongo-tools-r4.1.6 ...
>>> Source configured.
>>> Compiling source in /var/tmp/portage/app-admin/mongo-tools-4.1.6/work/mongo-tools-r4.1.6 ...
fatal: not a git repository (or any of the parent directories): .git
 * ERROR: app-admin/mongo-tools-4.1.6::local failed (compile phase):
 *   build failed
 *
 * Call stack:
 *     ebuild.sh, line 124:  Called src_compile
 *   environment, line 448:  Called die
 * The specific snippet of code:
 *       ./build.sh "${myconf[@]}" || die "build failed"
```

and thus i haven't tested mongodb itself. Please also update the pkg_pretend() section of mongodb to reflect the changes.

**Comment by okias** [comment]
@lKk2 how it's going? new MongoDB would be nice :+1:

**Comment by ultrabug** [comment]
I've bumped latest 3.6.12 and 4.0.8, we will not add unstable versions to portage, closing

Thanks everyone!

**Pull request closed without merging**